src/UserService.Validation/User/CreateUserRequestValidator.cs

```diff
@@ -2,50 +2,57 @@
 using LT.DigitalOffice.UserService.Models.Dto;
 using LT.DigitalOffice.UserService.Validation.User.Interfaces;
 using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace LT.DigitalOffice.UserService.Validation.User
 {
     public class CreateUserRequestValidator : AbstractValidator<CreateUserRequest>, ICreateUserRequestValidator
     {
+        private static Regex NameRegex = new(@"\d");
         public CreateUserRequestValidator()
         {
             RuleFor(user => user.FirstName)
                 .NotEmpty()
-                .MaximumLength(32).WithMessage("First name is too long.")
-                .MinimumLength(1).WithMessage("First name is too short.")
-                .Matches("^[A-Z][a-z]+$|^[А-ЯЁ][а-яё]+$").WithMessage("First name with error.");
+                .Must(x => !NameRegex.IsMatch(x))
+                .WithMessage("First name has incorrect format")
```

Comment:
Mb u should write it more specifically like “First name mustn’t have numbers”

Reply:
done